Src/Particle/AMReX_ParticleContainerI.H

```diff
@@ -995,7 +995,7 @@ ParticleContainer_impl<ParticleType, NArrayReal, NArrayInt, Allocator>::
 copyParticles (const PCType& other, bool local)
 {
     using PData = ConstParticleTileData<typename ParticleType::StorageParticleType, NArrayReal, NArrayInt>;
-    copyParticles(other, [=] AMREX_GPU_HOST_DEVICE (const PData& /*data*/, int /*i*/) { return 1; }, local);
+    copyParticles(other, [] AMREX_GPU_HOST_DEVICE (const PData& /*data*/, int /*i*/) { return 1; }, local);
```

Comment:
@WeiqunZhang should those be only `AMREX_GPU_DEVICE`?

Reply:
It should not matter. Both should work. If we can go back in time, we should provide AMREX_GPU_HOST_DEVICE only so that we can always detect extended lambda's function signature.

Reply:
not tried yet, but seems not needed